src/StudyRegistration/utils.tsx

```diff
@@ -35,13 +35,24 @@ export const preprocessStudyRegistrationMetadata = async (username, metadataID,
     metadataToUpdate[STUDY_DATA_FIELD][studyRegistrationTrackingField] = username;
 
     // add all repository_study_ids as separate objects
+    var tempStudyIDObj = [{}]
```

Comment:
```suggestion
    let tempStudyIDObj = [];
```

Reply:
Omitting the curly braces is giving a type error on the new `tempStudyIDObj`. Retaining the curly braces makes the error go away.